# Admittance parameters for two mass damper systems
# (the arm and the platform) with params (mass_arm, damping_arm)
# and (mass_platform, damping_platform) respectively, connected by a
# spring-damper with parameters (damping_coupling, stiffness_coupling,
# equilibrium_point_spring)
mass_arm: [6.0,0,0,0,0,0,
           0,6.0,0,0,0,0,
           0,0,6.0,0,0,0,
           0,0,0,4.0,0,0,
           0,0,0,0,4.0,0,
           0,0,0,0,0,4.0]

#damping_arm: [40.0,0,0,0,0,0,
#              0,40.0,0,0,0,0,
#              0,0,40.0,0,0,0,
#              0,0,0,40.0,0,0,
#              0,0,0,0,40.0,0,
#              0,0,0,0,0,40.0]

damping_arm: [0.75,0,0,0,0,0,
              0,0.75,0,0,0,0,
              0,0,0.75,0,0,0,
              0,0,0,0.75,0,0,
              0,0,0,0,0.75,0,
              0,0,0,0,0,0.75]

damping_null: [1.0,0,0,
               0,1.0,0,
               0,0,1.0]

DAMPING_INJECTION: false

REAL_ROBOT: false

